api/serializers.py

```diff
@@ -48,6 +62,18 @@ class Meta:
         extra_kwargs = {"password": {"write_only": True}}
 
 
+def read_video(video):
+    """
+    @param video: A FieldFile representing the video.
+    Commonly found on Video.video_file
+    """
+    video.open()
+    try:
+        return video.read().decode()
+    finally:
+        video.close()

```

Comment:
```suggestion
    try:
        return video.read().decode()
    finally:
        video.close()
```
You shouldn't need to explicitly open the file because [Django will open it for you](https://docs.djangoproject.com/en/3.1/ref/models/fields/#django.db.models.fields.files.FieldFile.open).